table/table.go

```diff
@@ -21,6 +21,11 @@ type Model struct {
 	styles Styles
 
 	viewport viewport.Model
+	renderedLines
+}
+
+type renderedLines struct {
+	start, end int
```

Comment:
I think we can get rid of this `renderedLines` struct and replace it as `end` and `start` on the model. What do you think?

Reply:
Absolutely fine with me, was just a distinction between original table struct and added values, pushed a commit with change